name-registry/README.md

```diff
@@ -0,0 +1,48 @@
+<p align="center">
+    <picture>
+        <source media="(prefers-color-scheme: dark)" srcset=".docs/fundraiser-logo-dark-theme.png">
+        <img alt="SwayApps Fundraiser Logo" width="400px" src=".docs/fundraiser-logo-light-theme.png">
+    </picture>
+</p>
+
+## Overview
+
+This is a simple name registry app built in Sway. Anyone can register a name for the price of 1 gwei per 100 seconds. The name will resolve to any Address/Identity that the owner sets.
```

Comment:
Fuel uses a precision of 9. I asked john and he said to use gwei

Reply:
I think we should avoid price comparisons and generally equating one chain to another. That is for some high level overview of the general protocol rather than building 1-to-1 comparisons. I suggest removing any reference to any concept of Ethereum, such as the price here, and coming up with your own abstraction e.g. create some asset and decide how much of that asset is required. The base asset could be such an asset.